test/import-http.bats

```diff
@@ -73,4 +73,34 @@ function teardown() {
     [ "$(sha reference/nm_orig)" == "$(sha dest/img/rootfs/root/nm)" ]
 }
 
+@test "importing to a dest" {
+    cat > img/stacker1.yaml <<EOF
```

Comment:
what do you gain by building 'stacker1' and 'stacker2' here rather than just one  thing?

Reply:
Was putting together a quick test. Fixed now.